types/websequencediagrams/index.d.ts

```diff
@@ -0,0 +1,19 @@
+// Type definitions for websequencediagrams 0.1
+// Project: https://github.com/hildjj/node-websequencediagrams
+// Definitions by: Florian Keller <https://github.com/ffflorian>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+/// <reference types="node" />
+
+export as namespace wsd;
```

Comment:
It seems like the `websequencediagrams` package is not a UMD module. If that's true, you don't need this line.